app/components/Form/ProjectMilestoneReportFormSummary.tsx

```diff
@@ -105,9 +105,31 @@ const ProjectMilestoneReportFormSummary: React.FC<Props> = ({
       // Set the formSchema and formData based on showing the diff or not
       const milestoneFormDiffObject = renderDiff
         ? getMilestoneFilteredSchema(
-            milestoneReport.formByJsonSchemaName.jsonSchema
-              .schema as JSONSchema7,
-            milestoneReport
+            {
+              ...(milestoneReport.formByJsonSchemaName.jsonSchema
+                .schema as JSONSchema7),
+              properties: {
```

Comment:
If you change the milestone type to Reporting, which doesn't have expenses, then the adjusted values still show up in the diffs: https://www.loom.com/share/8a62e10419a14496b6a7dcd67ffdf8fa. Do you think that was introduced by this cloning strategy? Also possible old things have always been hanging out in the new form data and we just didn't know until know because we never displayed diffs

Reply:
https://github.com/bcgov/cas-cif/issues/1745